mex/editor/edit/models.py

```diff
@@ -7,18 +7,21 @@
 class InputConfig(rx.Base):
     """Model for configuring input masks."""
 
-    badge_options: list[str] = []
-    editable_href: bool = False
-    editable_badge: bool = False
-    editable_text: bool = False
+    badge_default: str | None = None  # value to pre-select in drop-down menu
+    badge_options: list[str] = []  # possible values to show in drop-down menu
+    badge_titles: list[str] = []  # title explaining the drop-drown choice
```

Comment:
I was confused by the comment and thought that each choice has an explanation. I hope this comment captures it better. 

```suggestion
    badge_titles: list[str] = []  # title for the collection of drop-drown choices
```